.github/workflows/public-repos-scala-steward.yml

```diff
@@ -11,5 +11,6 @@ jobs:
     uses: ./.github/workflows/reusable-scala-steward.yml
     with:
       app_id: 214238
+      app_installation_id: 26822732
```

Comment:
One surprise here is that moving from [tibdex/github-app-token](https://github.com/tibdex/github-app-token) to [scala-steward-org/scala-steward-action](https://github.com/scala-steward-org/scala-steward-action) for App authentication, we now need to also supply the `github-app-installation-id`, where previously we didn't.

In the Scala Steward action, both ['app' & 'installation' tokens seem to get used](https://github.com/scala-steward-org/scala-steward-action/blob/190e7f36cd057dfb5d1b6fde688e6399ef249560/src/action/main.ts#L29-L35) - so far as I can see, the 'app' is used for performing actions as the bot, while 'installation' is used for setting up the Scala Steward workspace and I think cloning the required GitHub repos.

In our case, because we own both the app and the installation, the distinction probably doesn't help us very much, but I imagine it's useful if you're an organisation like VirtusLab, who run Scala Steward on other organisations repositories as a service.